GameData/Kerbalism/Profiles/Default.cfg

```diff
@@ -48,6 +48,14 @@ Profile
     refill_message = $VESSEL oxygen reserves restored@<i>The crew is taking a breather</i>
   }
 
+  Supply
+  {
+    resource = CarbonDioxide
```

Comment:
This may get spammy.

Reply:
It's not really, either your ship doesn't store CO2 and you don't get any messages about CO2 (even if you convert the CO2 on the spot into water again). Or you do store CO2 and you care. Or you are in the steady state of empty, which produces the message once, and then it stops. Theoretically you can have non-steady state consumption that frequently crosses the threshold, but that would affect every other resource as well, and is pretty rare.

Reply:
Not to mention that listing as Supply is neccesary for it to show up in the planner.